charts/sauron/newman/e2e_test.sh

```diff
@@ -1,11 +1,32 @@
 #!/bin/bash
 
+OUTPUT_DIR=$1
+if [[ $OUTPUT_DIR == "" ]]; then
+  echo "Empty output location. Please provide an output location when running this script. EX: ./e2e_tests.sh /sauron"
+  exit 1
+fi
```

Comment:
Needed for permission issues in sauron. Can't write to /usr/local/bin, so we must specify the mounted pvc (/sauron)